include/rocksdb/options.h

```diff
@@ -1349,6 +1349,15 @@ struct WriteOptions {
   // Default: false
   bool low_pri;
 
+  // If true, this writebatch will maintain the last insert positions of each
+  // memtable as hints in concurrent write. It can improve write performance
+  // in concurrent writes if keys in one writebatch are sequential. In
+  // non-concurrent writes (when concurrent_memtable_writes is false) this
+  // option will be ignored.
+  //
+  // Default: false
+  bool hint_per_batch;
```

Comment:
I think we should call it `memtable_insert_hint_per_batch` or something like that. Hint is too general for WriteOptions.

Reply:
I will change it to `memtable_insert_hint_per_batch `.